Management/src/Steeltoe.Samples.Management.sln

```diff
@@ -0,0 +1,36 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why was the leading line break removed in the last commit?

Reply:
Is there an important purpose to it? I actually removed it because I didn't expect it to be there... looking at our other samples, I don't see consistency

Reply:
It is how Visual Studio creates it. I don't think we should change it unless we have a reason to.